webapp/src/dogma/features/project/settings/mirrors/MirrorForm.tsx

```diff
@@ -242,7 +242,11 @@ const MirrorForm = ({ projectName, defaultValue, onSubmit, isWaitingResponse }:
               rules={{ required: true }}
               control={control}
               render={({ field: { onChange, value } }) => (
-                <RadioGroup onChange={onChange} value={value} defaultValue={defaultValue.direction}>
+                <RadioGroup
+                  onChange={onChange}
+                  value={value}
+                  defaultValue={defaultValue.direction || 'REMOTE_TO_LOCAL'}
```

Comment:
Since I'm not an expert, could you let me know the differences?
Previously, we set the default value for the schedule int the input component:
https://github.com/line/centraldogma/blob/ac4712b8bb551560ea7dd3f1e225f930445efae1/webapp/src/dogma/features/project/settings/mirrors/MirrorForm.tsx#L201

Reply:
It seems a cruft because `defaultValue.schedule` is not null.
I realized that the default values are set: 
https://github.com/line/centraldogma/blob/24603eb149d3945fde2cc1ced71a9efdcbc231fe/webapp/src/pages/app/projects/%5BprojectName%5D/settings/mirrors/new/index.tsx#L34-L48
Should we set the values there?

Reply:
Oh that works. Thanks! 😉